src/components/checkbox-group/checkbox/bl-checkbox.css

```diff
@@ -0,0 +1,67 @@
+:host {
+    display: inline-block;
+
+    --bg-color: var(--bl-checkbox-bg-color, var(--bl-color-primary));
```

Comment:
As we discussed in https://github.com/Trendyol/baklava/discussions/164 we don't define CSS variable in the host anymore. You can define in box class scope.

Reply:
fixed